man/docker-daemon.8.md

```diff
@@ -0,0 +1,402 @@
+% DOCKER(1) Docker User Manuals
+
+% William Henry 
+
+% SEPTEMBER 2015
+
+# NAME
+docker-daemon - Enable daemon mode
```

Comment:
Just a suggestion (open to better alternatives);

    docker-daemon - Run (/ Start) Docker in daemon mode